app/metrics.yaml

```diff
@@ -509,6 +509,220 @@ metrics:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
 
+preferences:
+  show_search_suggestions:
+    type: string_list
+    description: >
+      Whether or not the user has search suggestions enabled
+      default: true
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  remote_debugging:
+    type: string_list
+    description: >
+      Whether or not the user has remote debugging enabled
+      default: false
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  telemetry:
+    type: string_list
+    description: >
+      Whether or not the user has telemetry enabled
+      default: true
+    send_in_pings:
+      - metrics
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/11118
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11211
+    notification_emails:
+      - fenix-core@mozilla.com
+    expires: "2020-09-01"
+  tracking_protection:
+    type: string_list
+    description: >
+      What type of enhanced tracking protection the user has enabled
+      default: "standard"
```

Comment:
Can you list the options here "standard" "strict" "custom", and also what "" means?

Reply:
Updated!